test/e2e/annotations/canary.go

```diff
@@ -806,7 +805,7 @@ var _ = framework.DescribeAnnotation("canary-*", func() {
 				Contains(canaryService)
 		})
 
-		ginkgo.It("should route requests evenly split between mainline and canary if canary weight is 50", func() {
+		ginkgo.It("should route requests split between mainline and canary if canary weight is 50", func() {
```

Comment:
I'd not remove `evenly` here, because it is still expected to be evenly. And if you just send large enough number of requests it would be pretty even.